src/__test__/changeStatus.test.js

```diff
@@ -0,0 +1,16 @@
+import { changeStatus, checkTaskStaus } from '../__mocks__/changeStatus.js';
+import addTask from '../__mocks__/addTask.js';
```

Comment:
- [ ] One of your tests are failing. Please let's fix it. :point_down: 

![b](https://user-images.githubusercontent.com/57562869/155602564-3ca0e5eb-4783-4d7f-b2b0-021dd7e03815.png)